tests/integration/test_backup_restore_new/test.py

```diff
@@ -1560,6 +1560,39 @@ def test_backup_all(exclude_system_log_tables):
     instance.query("DROP USER u1")
 
 
+@pytest.mark.parametrize("include_database_name", [False, True])
+def test_backup_database_except(include_database_name):
+    create_and_fill_table()
+
+    session_id = new_session_id()
+    instance.query(
+        "CREATE TABLE test.omit_table (s String) ENGINE = MergeTree ORDER BY s",
+    )
+
+    omit_table_name = "test.omit_table" if include_database_name else "omit_table"
+    backup_name = new_backup_name()
+    backup_command = (
+        f"BACKUP DATABASE test EXCEPT TABLES {omit_table_name} TO {backup_name}"
+    )
+
+    instance.http_query(backup_command, params={"session_id": session_id})
```

Comment:
why do you use `instance.http_query()` and not `instance.query()` here?

Reply:
I just copied one of the tests, not sure if `session_id` is important.